wiki/Game_mode/de.md

```diff
@@ -1,3 +1,9 @@
+---
+stub: true
```

Comment:
Is there a reason why only the translations are marked as stubs?

Reply:
Not really, will change

Reply:
done